src/System.Runtime.Extensions/ref/System.Runtime.Extensions.cs

```diff
@@ -634,6 +634,7 @@ public static partial class Path
         public static string GetTempPath() { return default(string); }
         public static bool HasExtension(string path) { return default(bool); }
         public static bool IsPathRooted(string path) { return default(bool); }
+        public static string GetRelativePath(string relativeTo, string path) { return default(string); }
```

Comment:
@ericstj has a PR out https://github.com/dotnet/corefx/pull/11272 which shows how to add these specifically for .NET Core. We will be handling things that are part of netstandard differently moving forward and the APIs will be added in a different place from where we actually build .NET Core so these projects will essentially be converted to .NET Core refs only.